### DISCUSSION TITLE
DEM out of bounds:

### DISCUSSION THREAD

**Discussion by JonasardoLundUniv** [davidfrantz/force#96] [Q&A] [answered]
Hi
In the log when correcting Level1 - >LEVEL2 (force-suite force-level2 ./OSKAR_PRM/Oland_L8_to_L2.prm) I get **DEM out of bounds:**
My area include an Island as well as the water between the Island and main land to get a larger area covered in order to run co-registration. Any suggestions? Remove negative altitudes?

**From log:**
/projects/eko/fs5/JA/FORCE/L8L1/191020/LC08_L1TP_191020_20200324_20200326_01_T1: DEM out of bounds: min -0.999000 max 0.016116. Elevation statistics failed! unable to compile topography.

Regards
/Jonas
![image](https://user-images.githubusercontent.com/81447806/119164446-bae10a00-ba5c-11eb-83c1-c4b040a144c7.png)

![image](https://user-images.githubusercontent.com/81447806/119164089-5756dc80-ba5c-11eb-9115-74f26b90f813.png)

**Comment by davidfrantz** [comment, marked as answer]
Hi Jonas,
the error message prints the elevation range of the DEM in km. I assume that ``DEM_NODATA`` doesn't match with the actual nodata value in the DEM image.
Cheers,
David

**Comment by davidfrantz** [comment]
Hi @JonasardoLundUniv ,
did this answer solve your problem? 
If so, please hit the ``answered`` button, which may be useful for other people having the same issue.
Cheers,
David

**Reply by JonasardoLundUniv** [reply]
Yes, thanks works OK now. I made a mistake with missing values.
/Jonas
